Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -1460,6 +1468,11 @@ static void verboseLog(PDFIncrementalLoader* incrementalLoader, uint64_t streame
     "}"_s;
 }
 
+Color PDFPluginBase::pluginBackgroundColor()
+{
+    return roundAndClampToSRGBALossy([CocoaColor grayColor].CGColor);
```

Comment:
Maybe we should store this in a `static NeverDestroyed`. It'll get called every time the PDF is painted right?